helm/values.yaml

```diff
@@ -0,0 +1,52 @@
+---
+replicaCount: 1
```

Comment:
If I knew nothing about our system, I would have no clue what this replica count is for. That's the reason why we need to arrange the config on meaningful parts. I'll suggest it in another comment.